server/src/main/java/org/opensearch/common/geo/GeoUtils.java

```diff
@@ -95,6 +95,12 @@ public class GeoUtils {
     /** rounding error for quantized latitude and longitude values */
     public static final double TOLERANCE = 1E-6;
 
+    public static final PointParser POINT_PARSER;
+
+    static {
+        POINT_PARSER = new PointParser(LONGITUDE, LATITUDE, true);
```

Comment:
why this is a static section? I think we can do variable declaration and initialization in one line

Reply:
The variable is used in static methods in GeoUtils.

Reply:
I mean why can't we declare it like this without static section:

```
public static final PointParser POINT_PARSER = new PointParser(LONGITUDE, LATITUDE, true);
```

Reply:
Got it. Personally, I find it easier to read by having all static variable initialization in a single block. Also easier to extend it later if it cannot be initialized in a single line.